prism/src/main/java/org/apache/falcon/resource/AbstractInstanceManager.java

```diff
@@ -465,7 +465,7 @@ public int compare(Instance i1, Instance i2) {
                             : i2.getCluster().compareTo(i1.getCluster());
                 }
             });
-        } else if (orderBy.equals("starttime")){
+        } else if (orderBy.equals("starttime") || orderBy.isEmpty()){
```

Comment:
This means we are ordering the results by startTime, by default. But, the current behavior is to order by instanceTime in descending order and this change breaks that.